services/base-service/models/sitefarm/api.js

```diff
@@ -120,7 +120,7 @@ router.get(
           admin: req.query.admin ? true : false,
         }
         doc = await expert_model.get(expertId, opts);
-        doc=expert_model.sanitize(doc);
+        doc=expert_model.subselect(doc);
```

Comment:
@rakunkel-ucd this would limit to 25 works.  We can set that to whatever.   Do you have an idea for limiting the results?